integration-msk-to-lambda/example.go

```diff
@@ -0,0 +1,27 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+func handler(event map[string]map[string][]map[string]string) {
```

Comment:
Is this the correct signature? I think there's an existing type for MSK events here: https://pkg.go.dev/github.com/aws/aws-lambda-go/events#KafkaEvent

Reply:
Fixed the code to the add Kafka events